Atomic/rpm_host_install.py

```diff
@@ -36,10 +36,24 @@ def _copyfile(selinux_hnd, src, dest):
             os.symlink(linkto, dest)
             return True
         else:
-            # we cannot use shutil.copy2() or shutil.copystat() here as it would override the
-            # security.selinux xattr.
-            shutil.copy(src, dest)
-            shutil.copymode(src, dest)
+            file_copied = False
+            # newer version of Skopeo/SELinux use the same SELinux context for f
+            if try_hardlink and selinux_hnd is not None:
+                src_label = selinux.getfilecon(src)
```

Comment:
Below is from https://selinuxproject.org/page/LibselinuxAPISummary

getfilecon getfilecon_raw | Wrapper for the xattr API - Get file context, and set *con to refer to it. Caller must free via freecon(3).
-- | --

Do we need to worry about freeing context here then?

Reply:
The selinux bindings should be handling that, so no.